xterm/websockets-lib-funcs.pl

```diff
@@ -97,4 +97,17 @@ sub cleanup_miniserv_websockets
 &unlock_file(&get_miniserv_config_file());
 }
 
+sub check_miniserv_parent_session_websocket
+{
+my ($sid) = @_;
+&foreign_require('acl');
+my %miniserv;
+&get_miniserv_config(\%miniserv);
+&acl::open_session_db(\%miniserv);
```

Comment:
You'll need to close the session DB, or else I don't think updates done by `miniserv` are guaranteed to be visible..

Reply:
Got it. Thanks.